app/commands/v2/import.rb

```diff
@@ -0,0 +1,98 @@
+module Commands
+  module V2
+    class Import < BaseCommand
+      def call
+        unless UuidValidator.valid?(payload[:content_id])
+          raise CommandError.new(
+          code: 422,
+          error_details: {
+            error: {
+              code: 422,
+              message: "Content id not valid",
+              fields: "content_id",
+            }
+          })
+        end
+
+        delete_all(payload[:content_id])
+        payload[:content_items].map.with_index do |event, index|
+          create_content_item(event, index, payload[:content_id])
+        end
+
+        after_transaction_commit do
+          send_downstream(payload[:content_items].find { |e| e[:action] == 'Publish' })
+        end
+
+        Success.new(content_id: payload[:content_id])
+      end
+
+    private
+
+      def send_downstream(content)
+        return unless content
+        DownstreamLiveWorker.perform_async_in_queue(
+          DownstreamLiveWorker::LOW_QUEUE,
+          content_id: content[:content_id],
+          locale: 'en',
+          message_queue_update_type: content[:payload][:update_type],
+          payload_version: event.id
+        )
+      end
+
+      def create_content_item(event, index, content_id)
+        event_payload = event[:payload]
+        content_item_payload = event_payload.except(:state)
+
+        validate_content_item_payload(content_item_payload)
+
+        Services::CreateContentItem.new(
+          payload: content_item_payload.merge(content_id: content_id),
+          user_facing_version: index + 1,
+          lock_version: index + 1,
```

Comment:
Probably not, but it seems to me like the not doing so would look odd in the database?